### PR TITLE
compute total row

### PR DESCRIPTION
**Pull request by nickpell**
:blowfish: 

@snopoke @czue 

Right now I'm allowing the query to fail if an attempt is made to calculate the total of a non-integer column.  If this turns out too difficult to validate at the UCR level, I may add some explicit type casting to the query.

### CODE REVIEW

File: sqlagg/base.py [hunk 1]
```diff
@@ -79,6 +79,23 @@ def count(self, metadata, connection, filter_values):
...
+        assert self.limit is None
+
+        def _generate_total_column(column_name, selectable):
+            from sqlagg import SumColumn
+            return SumColumn(column_name).sql_column.build_column(selectable)
+
+        subquery = self._build_query(metadata).alias()
+        query = sqlalchemy.select().select_from(subquery)
+        for total_column in total_columns:
+            query.append_column(_generate_total_column(total_column, subquery))
```
Comment (snopoke) [inline sqlagg/base.py]: what does this query end up actually looking like? Is it like this:
```sql
select sum(a), sum(b) from (
    select c as a, d as b from some_table where ....
)
```

**Comment by snopoke** [comment]
Looks ok. No tests though.

File: sqlagg/base.py (hunk 1 above)
Comment (nickpell) [inline sqlagg/base.py]: that's correct

**Comment by snopoke** [comment]
Waiting for tests unless this is time sensative

**Comment by nickpell** [comment]
added tests

**Comment by czue** [comment]
:+1: thanks

**Pull request merged**